std/experimental/ndslice/slice.d

```diff
@@ -1767,6 +1767,183 @@ struct Slice(size_t _N, _Range)
         assert(a != [[9, 2], [3, 4]]);
     }
 
+    /++
+    Computes hash value using MurmurHash3 algorithms without finalization step.
+    Hash map implementations should care about finalization step.
+    Built-in associative arrays has finalization step.
+
+    See_also: $(LREF Slice.toMurmurHash3), $(MREF std, _digest, murmurhash)
+    +/
+    size_t toHash() const
```

Comment:
Ah, the `struct` itself is a template, nvm.
So now I'm left wondering why all those `unittest` are nested :)